include/z64.h

```diff
@@ -452,6 +452,14 @@ typedef struct {
 
 typedef void(*fault_update_input_func)(Input* input);
 
+typedef enum {
+    /* 0x00 */ PAUSE_ITEM,
+    /* 0x01 */ PAUSE_MAP,
+    /* 0x02 */ PAUSE_QUEST,
+    /* 0x03 */ PAUSE_MASK,
+    /* 0x04 */ PAUSE_WORLD_MAP
+} PauseMenuPage;
```

Comment:
Wouldn't be better to put  this enum on a kaleido header and include that header instead?

Reply:
I kept it in z64.h for now because that's where `PauseContext` currently resides, and that's the struct that primarily uses this enum is. Is the intent for all the `Contexts` imbedded in `PlayState` to go into their own subheader as well?

Reply:
Yes, at some point we will move all the states structs to their own headers, like `SelectContext` does currently.
Since `PauseContext` is already on `z64.h` I guess is fine to keep this enum here for now.

Reply:
So I moved `PauseMenuPage` to the kaleido header, but I'll save moving `PauseContext` there for another PR